### ISSUE TITLE
Update copper.sh?

### ISSUE THREAD

**Issue by amitsaha** [cloud66-oss/copper#22]
Hi all, it looks like https://copper.sh still refers and documents he v1 of copper instead of the current V2 version?

**Comment by bravocharlie007** [comment]
That is correct. I don't understand how to install v2.0.1. Should it be installed using rubygems as well? How do I get my pc to recognize and see it?

**Comment by igoragueme** [comment]
> That is correct. I don't understand how to install v2.0.1. Should it be installed using rubygems as well? How do I get my pc to recognize and see it?


I have the same doubt.
Even running the command "copper update" copper seems to continue with the same version.

**Comment by khash** [comment]
You are correct: copper.sh is the old website and will be removed. You can find Copper v2 here https://github.com/cloud66-oss/copper/releases/tag/2.0.1
